pkgs/development/python-modules/krb5/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, cython
+, fetchFromGitHub
+, k5test
+, pkgs
+, pytestCheckHook
+, pythonOlder
+, which
+}:
+
+buildPythonPackage rec {
+  pname = "krb5";
+  version = "0.4.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "jborean93";
+    repo = "pykrb5";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-aifu99CDT/RtDmqIK3SAQATDCvbR9FvwRR2L3hYG0gs=";
+  };
+
+  nativeBuildInputs = [
+    cython
+    pkgs.krb5
```

```suggestion
    libkrb5
```
could be an option around passing in pkgs